.github/workflows/build-and-deploy.yml

```
@@ -521,6 +528,47 @@ jobs:
           custom-secret-0: ${{ secrets.AUR_USERNAME }}
           custom-secret-1: ${{ secrets.AUR_EMAIL }}
           custom-secret-2: ${{ secrets.AUR_SSH_PRIVATE_KEY }}
+
+  # Completes the release process for winget
+  release-winget:
+    needs:
+      - ci-preparation
+      - release
+    concurrency:
+      group: release-winget-${{ needs.ci-preparation.outputs.build-ref }}
+      cancel-in-progress: false
+    runs-on: windows-2022
+    env:
+      packageID: "Unibo.Alchemist"
+    if: >-
+      !github.event.repository.fork
+      && (
+        github.event_name != 'pull_request'
+        || github.event.pull_request.head.repo.full_name == github.repository
+      )
+      && fromJSON(needs.release.outputs.winget-deploy)
```

```suggestion
    if: fromJSON(needs.release.outputs.winget-deploy)
```

If I understand the workflow correctly, this variable will be true only after a successful release, which in turn means that this condition alone means that we have just released on the main branch.